### ISSUE TITLE
Issues during the reproduction process

### ISSUE THREAD

**Issue by bgnbl** [ActiveVisionLab/SD4Match#5]
I encountered an error while running pre_computate-dino_feature. py:ValueError: Unable to create tensor, you should probably activate padding with 'padding=True' to have batched tensors with the same length.I added padding=True in： img=vision-processor (img, return_tensors="pt", padding=True) ["pixel_values"], but it still reported an error.

**Comment by Xinghui-Li** [comment]
Hi, 

I am sorry that I have not encountered this problem before. I have googled it and a possible reason could be incompatibility of package. Would you mind trying to follow [this post](https://stackoverflow.com/questions/75623118/valueerror-unable-to-create-tensor-issue-for-a-transformer-model) by rolling back numpy to a version earlier than 2.0? and please tell me whether this is working or not afterward.

**Comment by bgnbl** [comment]
Hi, Thank you for your help, yesterday I tried downgrading the version of NumPy to a version earlier than 2.0, and it worked fine. However, I encountered another issue, which is: torch.distributed.elastic.multiprocessing.errors.ChildFailedError. I am wondering if it might be because I don't have enough GPUs.


________________________________
发件人: Xinghui-Li ***@***.***>
发送时间: 2024年12月16日 1:41
收件人: ActiveVisionLab/SD4Match ***@***.***>
抄送: bgnbl ***@***.***>; Author ***@***.***>
主题: Re: [ActiveVisionLab/SD4Match] Issues during the reproduction process (Issue #5)


Hi,

I am sorry that I have not encountered this problem before. I have googled it and a possible reason could be incompatibility of package. Would you mind trying to follow this post<https://stackoverflow.com/questions/75623118/valueerror-unable-to-create-tensor-issue-for-a-transformer-model> by rolling back numpy to a version earlier than 2.0? and please tell me whether this is working or not afterward.

―

**Comment by Xinghui-Li** [comment]
Yes, please check whether your number of GPUs matches the configuration in the bash script. The --num_processes should match the number of GPUs.

**Comment by bgnbl** [comment]
When running the script, I encountered an error at Spair-7k indicating that the file SD4Match-main/asset/sd4match/log/spair/CPM_spair_sd2-1_Pair-DINO-Feat-G25-C50_constant_lr0.01/pytorch_model.bin is missing. Do I need to download this weight from Hugging Face myself, or is it something that shouldn't need to be provided?


________________________________
发件人: Xinghui-Li ***@***.***>
发送时间: 2024年12月17日 3:03
收件人: ActiveVisionLab/SD4Match ***@***.***>
抄送: bgnbl ***@***.***>; Author ***@***.***>
主题: Re: [ActiveVisionLab/SD4Match] Issues during the reproduction process (Issue #5)


Yes, please check whether your number of GPUs matches the configuration in the bash script. The --num_processes should match the number of GPUs.

―
